WebHostLib/upload.py

```diff
@@ -104,13 +104,21 @@ def upload_zip_to_db(zfile: zipfile.ZipFile, owner=None, meta={"race": False}, s
 
         # Factorio
         elif file.filename.endswith(".zip"):
-            _, _, slot_id, *_ = file.filename.split('_')[0].split('-', 3)
+            try:
+                _, _, slot_id, *_ = file.filename.split('_')[0].split('-', 3)
+            except Exception:
```

Comment:
pycharm should give you a warning for that `except` because it's too broad.
```suggestion
            except ValueError:
```